lib/class_core.php

```diff
@@ -740,7 +740,7 @@ function save_property( $post_id = null ) {
     ) );
 
     // Checking if this is a child property if then add it to children array.
-    if($parent_id = wp_get_post_parent_id($post_id)){
+    if($prev_parent_id = wp_get_post_parent_id($post_id)){
```

Comment:
Renamed to ensure that it not get replaced as we are passing this with 'save_property'.